xml/System.Data/ParameterDirection.xml

```diff
@@ -189,7 +189,7 @@
       </ReturnValue>
       <MemberValue>6</MemberValue>
       <Docs>
-        <summary>The parameter represents a return value from an operation such as a stored procedure, built-in function, or user-defined function.</summary>
+        <summary>The parameter represents a return value from an operation such as a stored procedure, built-in function, or user-defined function.  Specifying a named parameter enables the <see cref="T:System.Data.IDbCommand" /> to locate and read the return value, even when data rows are also returned.</summary>
```

Comment:
* How is the parameter being named specifically relevant here? Also, named and non-named parameter are once again a database-specific detail (some databases don't support named parameters at all).
* We generally discourage use of the ADO.NET interfaces (e.g. `IDbCommand`), which have been largely replaced by the abstract base classes (`DbCommand`). This is because the interfaces could not be evolved with new APIs (e.g. async execution) without breaking existing implementation.

In general, the existing text here seems sufficient to me.